### ISSUE TITLE
Utility Request: background-repeat 

### ISSUE THREAD

**Issue by pxwee5** [tailwindcss/tailwindcss#247]
I believe the background-repeat utility is missing. Any specific reason why it is not included?

**Comment by adamwathan** [comment]
No specific reason, just haven't gotten around to discussing it yet. What values do you think are smart to include by default? I'm thinking at least `bg-repeat` and `bg-no-repeat` but not sure if the others are commonly used enough to justify including by default.

**Comment by benface** [comment]
I would like at least `bg-repeat-x` and `bg-repeat-y`. I personally don't care about the `space` and `round` values and don't think they're commonly used.

**Comment by pxwee5** [comment]
Well, for starters it is a fail safe when used with `background-position: center`.

Also in practise, a quick way to always make your content responsive for all screen sizes were `background: no-repeat center / cover`, which translates to:
```
background-repeat: no-repeat;
background-position: center;
background-size: cover; // or contain
```

**Comment by sorinr** [comment]
The implicit value is repeat so bg-repeat its not so useful. In a situation when you dynamically want to change a background (bg-no-repeat) into a repeat you just remove that class.  I agree with @benface for the `bg-repeat-x` and `bg-repeat-y`

**Comment by adamwathan** [comment]
> The implicit value is repeat so bg-repeat its not so useful.

Actually this means it's mandatory to include if we have any background-repeat utilities, otherwise there is no way to get back to a default state at larger screen sizes.

So I think the four we should add are:

```
bg-repeat
bg-no-repeat
bg-repeat-x
bg-repeat-y
```

Happy to accept a PR for getting this started, otherwise I can start something if I have time over the next few days.

**Comment by sorinr** [comment]
@pxwee5 as you initiated it do u have time in making that PR?

**Comment by MichaelDeBoey** [comment]
@sorinr @pxwee5 PR is on it's way 🙂😉

**Comment by MichaelDeBoey** [comment]
@sorinr @pxwee5 PR is merged with the `0.3` branch now 🙂

**Comment by adamwathan** [comment]
Closing since this is now in 0.3:

https://tailwindcss.com/docs/background-repeat
